manifests/e/Eliot/UEExplorer/1.3.1/Eliot.UEExplorer.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Eliot.UEExplorer
+PackageVersion: 1.3.1
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86

```

Comment:
```suggestion
  Architecture: x64
```
Noticed the architecture in Task Manager. Do we need to change this here?
![image](https://user-images.githubusercontent.com/88161975/215199598-72119cda-1e06-4e80-8817-f5e38db3c167.png)

Reply:
Strange, I compiled it targetting x86:
![image](https://user-images.githubusercontent.com/808593/215201279-6a04386d-1768-4b1f-a691-72a3bf842389.png)